app/Commands/StopCommand.php

```diff
@@ -27,11 +26,19 @@ public function handle(Docker $docker, Environment $environment): void
         $this->environment = $environment;
         $this->initializeCommand();
 
-        $container = $this->argument('containerId');
+        $containers = $this->argument('containerId');
 
-        if ($container) {
-            $this->stop($container);
+        if (filled($containers)) {
+            foreach ($containers as $container) {
+                $this->stop($container);
+            }
+            return;
+        }
 
+        if ($this->option('all')) {
+            foreach (app(Docker::class)->stoppableTakeoutContainers() as $stopableContainer) {
```

Comment:
Just one small change; could you rename this variable to `stoppableContainer`?

Thanks!

Reply:
There is also an instance of docker already available 
$this->docker->
😁

Reply:
Fixed! @mattstauffer